kafka-spring-boot-autoconfigure/src/test/java/org/axonframework/extensions/kafka/autoconfig/KafkaAutoConfigurationTest.java

```diff
@@ -136,32 +187,32 @@ void testAutoConfigurationWithMinimalRequiredProperties() {
     @Test
     void testConsumerPropertiesAreAdjustedAsExpected() {
         this.contextRunner.withUserConfiguration(TestConfiguration.class)
-                          .withPropertyValues(
```

Comment:
I am afraid this indentation change is not in line with the style guide. Would you mind verifying whether you're using [this](https://github.com/AxonFramework/AxonFramework/blob/master/axon_code_style.xml) style for this PR?